docs/source/release_notes.rst

```diff
@@ -6,7 +6,7 @@
 Releases
 ========
 
-Version 1.8.0-beta.1
+Version 1.8.0
```

Comment:
I don't think this causes an RST issue, but the under line is longer than the text, I know when the underline is shorter than the text it causes an issue.